utils/threshAlgos.py

```diff
@@ -1035,6 +1035,16 @@ def sbitRateAnalysis(chamber_config, rateTree, cutOffRate=0.0, debug=False, outf
     dataPath = getDataPath()
     elogPath = getElogPath()
 
+    # Map TPad to correct vfat
+    from ..mapping.chamberInfo import chamber_vfatPos2PadIdx
+    from tabulate import tabulate
+
+    from gempython.gemplotting.utils.anautilities import findInflectionPts
+    from gempython.gemplotting.utils.anautilities import make3x8Canvas
+
+    # Allow for yellow warning color
+    from gempython.utils.gemlogger import printYellow
```

Comment:
I would have these import statements be moved to where they are first used...this improves readability so you don't have to go through the code to understand where something comes from/what something is.

Reply:
I do (and prefer) the opposite, sort of following `c/c++` style, but at the scope of relevance (`class`, `function`, file)...
[PEP8](https://www.python.org/dev/peps/pep-0008/#imports)  says they should *always* be at the top of the file, and as there isn't an established "style" here, it can be left up to the individual developer, with a slight lean towards the PEP

Reply:
this comment can be ignored then...would suggest that this type of style guide info makes it into the manifesto.